python/MuonPhotonMatch_cff_miniAOD.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+# JMTBAD muon-photon matching is done here using the default muon
+# momentum and not whichever refit momentum will be eventually used in
+# the analysis. For most muons this shouldn't make much difference as
+# the matching is in eta-phi, but if the default is really bad and the
+# selected refit manages to recover, the photon match may be
+# wrong. The photon brem recovery needs to be studied more anyway.
+muonPhotonMatch = cms.EDProducer('TrivialDeltaRViewMatcher',
+                                 src     = cms.InputTag('slimmedMuons'),
+                                 matched = cms.InputTag('slimmedPhotons'),
+                                 distMin = cms.double(0.1)
+                                 )
+
+def addUserData(patMuonProducer, tag=cms.InputTag('muonPhotonMatch')):
+    patMuonProducer.userData.userCands.src.append(tag)
```

Comment:
Is there a chance of confusion with the muonPhotonMatch in python/MuonPhotonMatch_cff.py of the same name?
why not use something like the following in this file:
import MuonPhotonMatch_cff.py
muonPhotonMatch_miniAOD = MuonPhotonMatch_cff.muonPhotonMatch.clone()
then change the src and matched values of muonPhotonMatch_miniAOD?